src/apps/testapps/testGridPathCells.c

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 /** @file
- * @brief tests H3 distance function.
+ * @brief tests H3 grid path function.
  *
- *  usage: `testH3Distance`
+ *  usage: `testGridPathCells`
```

Comment:
I'll let you two hash that out. I don't have a strong view either way on this.

Reply:
How about we just call this `gridPath` with (`Cell` being implicit), while still having `gridPathEdges` and `gridPathDirectedEdges` (if we ever do write those functions).

I like `gridPath` more than `gridLine`, just because it aligns with a "path" in graph theory. If I didn't know what this function did, I think I'd be more likely to guess correctly with `gridPath`, but that's just my take. I'm not sure what the "average" user would guess.

Reply:
Sure, I think that's an improvement.

Reply:
We can have a followup PR on this including the RFC change?

Reply:
Will do!

Reply:
https://github.com/uber/h3/pull/441

I also added an open question around if we should do a similar thing for `compact` and `uncompact`.

Reply:
I'd vote for you to land this PR as-is, and I can take a stab at implementing whatever we decide for #441 in a follow-up PR.